### PR TITLE
chore: deprecate legacy features on x/token,collection

### PR DESCRIPTION
**Pull request by 0Tech**
## Description

This patch would deprecate following features on original modules:
- x/token
  - _all features_
- x/collection
  - fungible tokens
  - non-fungible tokens hierarchy

## Motivation and context

The features would be wiped out from the sdk in the next major release.

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/finschia-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/finschia-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

**Review by tkxkd0159 — DISMISSED** [review]
LGTM

File: proto/lbm/collection/v1/tx.proto [hunk 1]
```diff
@@ -89,15 +97,19 @@ service Msg {
...
+  }
 
   // OperatorBurnFT defines a method to burn fungible tokens of the holder by the operator.
   // Fires:
   // - EventBurned
   // - burn_ft_from (deprecated, not typed)
   // - burn_nft_from (deprecated, not typed)
   // - operation_burn_nft (deprecated, not typed)
-  rpc OperatorBurnFT(MsgOperatorBurnFT) returns (MsgOperatorBurnFTResponse);
+  rpc OperatorBurnFT(MsgOperatorBurnFT) returns (MsgOperatorBurnFTResponse) {
```
Comment (zemyblue) [inline proto/lbm/collection/v1/tx.proto:110]: `MsgOperatorBurnFT` message is not deprecated.

**Review by zemyblue — COMMENTED** [review]
Please update swagger also.

**Comment by 0Tech** [comment]
> Please update swagger also.

It seems the change does not affect the output for swagger.

**Review by zemyblue — APPROVED** [review]

**Review by tkxkd0159 — APPROVED** [review]

**Pull request merged**